sanityv3/schemas/editors/titleEditorContentType.tsx

```diff
@@ -1,14 +1,43 @@
 import { SuperScriptRenderer, SubScriptRenderer, StrikeThroughRenderer } from '../components'
-import { IconSuperScript, IconSubScript } from '../../icons'
+import { IconSuperScript, IconSubScript, EdsBlockEditorIcon } from '../../icons'
 import { StrikethroughIcon } from '@sanity/icons'
 import type { BlockFieldType } from '../../types/schemaTypes'
+import { format_color_text } from '@equinor/eds-icons'
+
+export type TitleContentProps = {
+  highlight?: boolean
+  extraLarge?: boolean
+  isBigTitle?: boolean
+  large?: {
+    title: string
+    value: 'normal'
+    component?: ({ children }: { children: React.ReactNode }) => JSX.Element
+  }
+}
 
 // TODO: Add relevant styles for titles (i.e. highlighted text)
-export const configureTitleBlockContent = (): BlockFieldType => {
-  // const { strong, emphasis, strikethrough } = options
-  return {
+export const configureTitleBlockContent = (options: TitleContentProps = {}): BlockFieldType => {
```

Comment:
I don't agree with having this logic here. This is a generic component, so IMO we should keep component specific logic out of this file. For example, "normal" text size label is always defaulting to "large", which is not the case.

Reply:
Suggestion:

```
export type TitleContentProps = {
  styles?: BlockStyleDefinition[]
  highlight?: boolean
}
```

and extract the new title logic into the sharedHeaderFields